packages/create-atomic/templates/src/templates/default.html

```diff
@@ -0,0 +1,110 @@
+<atomic-result-template>
```

Comment:
I thought that splitting the templates from the index.html would be much cleaner